decentraland/unity-renderer
# feat: enable channel links for everyone

## What does this PR change?

`Fixes #3953 `

Enable channel links for everyone. You will be able to join only if already exists.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=feat/enable-channel-links
2. Write in the chat any existing channel, for example #english. It should be highlighted and clickable. When you click on the channel link, it should show a prompts asking you to join it. The join should work and the channel should be opened.
3. Write in the chat any non-existing channel, for example #non-existing-channel. When you click on the channel link, it should show a prompts asking you to join it. You will not be able to join it, instead an error toast should appear at the top of the screen.
4. If you click on a channel link which you already joined, the channel window should be opened automatically.
5. By writing chat links in the chat try #nearby channel, custom channels and private chats.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
